package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "@emotion/react": "^11.6.0",
```

Comment:
I believe we need to do "npm install @emotion/react" (nts--add this to readme later)